lib/tasks/apipie_dsl.rake

```diff
@@ -0,0 +1,18 @@
+desc 'Apipie DSL cache specific tasks'
+namespace :apipie_dsl do
+  desc 'Generate cache index for all languages (override with FOREMAN_APIPIE_LANGS environment variable)'
```

Comment:
Per some similar conversations, should we default to English and let users choose if they want other languages for efficiency when generating?

Reply:
I agree, I think that API documentation should always be English only. I mean, a programmer that can't understand English is not going to be very efficient...